.travis.yml

```diff
@@ -10,10 +10,6 @@ env:
   jobs:
     - TEST_SUITE=''
```

Comment:
We can also eliminate this bit (and the unquoted `${TEST_SUITE}` below) -- it was introduced in 788758ba8 to support the Pirlo suite.

Reply:
Is that also true for the `${TEST_SUITE} added in 788758ba8db6bf6c5fab98b78c729c71d829b39d?